src/game/Object/DynamicObject.cpp

```diff
@@ -226,3 +226,8 @@ bool DynamicObject::IsFriendlyTo(Unit const* unit) const
     else
         { return true; }
 }
+
+void DynamicObject::SendObjectDeSpawnAnim()
+{
+    //TODO find a way since SMSG_GAMEOBJECT_DESPAWN_ANIM does not fit
+}
```

Comment:
Maybe, though I doubt it. I neither experimented with it nor have read the handlers in more details. The only thing I know for sure, that the SMSG_GAMEOBJECT_DESPAWN_ANIM gives no effect here, because that handler requires strictly gameobject GUID.